bin/render-stylesheet.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+const { renderSync } = require( 'sass' );
+const fs = require( 'fs' );
+const packageImporter = require( 'node-sass-package-importer' );
+
+if ( ! process.argv[ 2 ] ) {
+	throw new Error( 'No input file provided.' );
+}
+
+if ( ! process.argv[ 3 ] ) {
+	throw new Error( 'No output file provided.' );
+}
+
+const { css } = renderSync( {
+	file: process.argv[ 2 ],
+	outputStyle: 'compressed',
+	includePaths: [ 'client' ],
```

Comment:
The `includePaths` option could be removed after patching just one import in `reader-mobile.scss`:
```
@import 'blocks/reader-full-post/_content';
```
The path is neither relative to the current directory, neither it refers to a package. It relies on a lookup in `client/`. Replacing it with
```
@import 'calypso/blocks/reader-full-post/_content';
```
makes it a package reference, and is in line with the package import fix (adding `calypso/` prefix) we recently did.

Reply:
Makes sense to me 👍